tsdb/index/index.go

```diff
@@ -1423,7 +1457,7 @@ func (r *Reader) LabelValues(names ...string) (StringTuples, error) {
 
 	skip := 0
 	for d.Err() == nil {
-		if skip == 0 {
+		if skip == 0 || true {
```

Comment:
@brian-brazil mistake?

Reply:
Test code that slipped through, I wanted to be sure I had one that was unsorted.